sdk/src/main/java/com/uber/sdk/rides/client/model/RideReceipt.java

```diff
@@ -0,0 +1,148 @@
+
+package com.uber.sdk.rides.client.model;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * A receipt for a completed request.
+ * See
+ * <a href="https://developer.uber.com/docs/v1-requests-receipt">Ride Request Receipt</a>
+ * for more information.
+ */
+public class RideReceipt {
+    private String request_id;
+    private List<Charge> charges;
+
+    @Nullable
+    private Charge surge_charge;
+
+    private List<Charge> charge_adjustments;
+    private String normal_fare;
+    private String subtotal;
+    private String total_charged;
+
+    @Nullable
+    private Float total_owed;
+
+    private String currency_code;
+    private String duration;
+    private String distance;
+    private String distance_label;
+
+    /**
+     * The unique ID of the ride.
+     */
+    public String getRequestId() {
+        return request_id;
+    }
+
+    /**
+     * The charges made against the rider.
```

Comment:
Most Javadoc style guides (e.g., http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html) say "Method descriptions begin with a verb phrase.", so I would change this to "Returns the charges made against the rider." and make similar adjustments to the method Javadocs below.